### ISSUE TITLE
Replace any empty file titles with file name

### ISSUE THREAD

**Issue by newmanld** [uclibs/scholar_uc#847]
With the closing of #409 it should not be possible going forward to attach a file with a blank file title, either on work submission or after a work is created.  We should look for any existing files with a blank file title and correct (through the console?) to be the same as file name.  It is an accessibility issue as referenced in #521, where if file title exists the download button should not be too general.

**Comment by newmanld** [comment]
This work could perhaps be done when we are also looking at #788

**Comment by hortongn** [comment]
@newmanld Just checking to see if this should be closed or if all the files without titles have been corrected.

**Comment by newmanld** [comment]
Done, except for 4 files that may not be there anymore.  Note sent to @hortongn  to confirm before closing.
